dbt/adapters/bigquery/impl.py

```diff
@@ -911,25 +922,22 @@ def python_submission_helpers(self) -> Dict[str, Type[PythonJobHelper]]:
         }
 
     @classmethod
-    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
-        if constraint.type == ConstraintType.not_null:
-            return super().render_column_constraint(constraint)
-        elif (
+    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> Optional[str]:
```

Comment:
If `ConstraintType.primary_key` and `ConstraintType.foreign_key` are not enforced, shouldn't that be reflected on lines 178-9 in `CONSTRAINT_SUPPORT`? And if so, and assuming I got my red/green left/right correct this time, is this also something we can put into `dbt-core`?